src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteDeployer/StopServer.ps1

```diff
@@ -0,0 +1,26 @@
+﻿[CmdletBinding()]
+param(
+	[Parameter(Mandatory=$true)]
+	[string]$serverProcessName,
+	
+	[Parameter(Mandatory=$true)]
+	[string]$serverType,
+
+	[Parameter(Mandatory=$true)]
+	[string]$serverName
+)
+
+Write-Host "Executing the stop server script on machine '$serverName'"
+
+if($serverType -eq "IIS")
+{
+	throw [System.NotImplementedException] "IIS deployment scenarios not yet implemented."
+}
+else
+{
+	Write-Host "Stopping the process '$serverProcessName'"
+	Stop-Process -Name "$serverProcessName"
```

Comment:
I think you should taskkill the process instead because it might have one or more child processes